specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkVirtualAppliance.json

```diff
@@ -0,0 +1,509 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/networkVirtualAppliances": {
+      "get": {
+        "tags": [
+          "NetworkVirtualAppliances"
+        ],
+        "operationId": "NetworkVirtualAppliances_List",
+        "description": "Gets all Network Virtual Appliances in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of Network Virtual Appliance resources.",
+            "schema": {
+              "$ref": "#/definitions/NetworkVirtualApplianceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all Network Virtual Appliances for a given subscription": {
+            "$ref": "./examples/NetworkVirtualApplianceListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkVirtualAppliance": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkVirtualAppliancePropertiesFormat",
+          "description": "Properties of the Network Virtual Appliance"
+        },
+        "virtualHub": {
+          "readOnly": false,
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The Virtual Hub where Network Virtual Appliance is being deployed"
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets a unique read-only string that changes whenever the resource is updated"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "NetworkVirtualAppliance Resource."
+    },
+    "NetworkVirtualAppliancePropertiesFormat": {
+      "properties": {
+        "bootStrapConfigurationBlob": {
+          "type": "array",
+          "readOnly": false,
+          "description": "BootStrapConfigurationBlob storage URLs",
+          "items": {
+            "type": "string"
+          }
+        },
+        "cloudInitConfigurationBlob": {
+          "type": "array",
+          "readOnly": false,
+          "description": "CloudInitConfigurationBlob storage URLs",
+          "items": {
+            "type": "string"
+          }
+        },
+        "identity": {
```

Comment:
Is this trying to use a managed identity? Based on the examples it looks like it. If so it should be a top level property as its managed by ARM.
https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/Doc.aspx?sourcedoc=%7B66505FCE-8057-4524-B84F-4CEFE2D8A77C%7D&file=ARMResourcesWithStrongIdentity.docx&action=default&mobileredirect=true

Reply:
Done